### PR TITLE
docs: Create basic story for < VisualDescription />, < ColorDescription />

### LINKED ISSUE #92: docs: Create basic story for < VisualDescription />, < ColorDescription />
[Monday ticket](https://monday.monday.com/boards/3532714909/pulses/5350388750)

Add basic story by structure similar to existing ones

- https://github.com/mondaycom/vibe-storybook-components/blob/master/CONTRIBUTING.md
- https://github.com/mondaycom/vibe-storybook-components/blob/master/COMPONENTS_DOCUMENTATION_GUIDELINES.md

### PR DESCRIPTION
**Pull request by Dhoni77**
Closes #92

- Add stories for `< VisualDescription />` and `< ColorDescription />`
- Configured storybook path to display image for `< VisualDescription />`

Attachments:

VisualDescription:

visual-description

ColorDescription:

color-description

### CODE REVIEW

File: src/components/color-description/color-description.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .vibe-sb-comps-color-description {
   border-radius: var(--sb-border-radius-extra-small);
+  width: auto;
```
Comment (SergeyRoyt) [inline src/components/color-description/color-description.scss:3]: For me it seems to work fine even without this chane

Comment (Dhoni77) [reply src/components/color-description/color-description.scss:3]: Ok will remove it then👍

File: src/components/color-description/__stories__/color-description.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import * as ColorDescription from './color-description.stories';
+
+<Meta of={ColorDescription} />
+
+# ColorDescription
+
+## Overview
+
+ColorDescription component is used to display a color description.
+It combines the VisualDescription component with a colored square to visually represent a specific color.
```
Comment (SergeyRoyt) [inline src/components/color-description/__stories__/color-description.mdx:11]: Link might help here
```suggestion
It combines the<Link href={"/?path=/docs/components-visualdescription--docs"}>VisualDescription</Link>component with a colored square to visually represent a specific color.
```

**Review by SergeyRoyt — CHANGES_REQUESTED** [review]
Few small fixes needed, the rest is great!

**Review by SergeyRoyt — APPROVED** [review]
Great job! Thanks a lot!! 👑

**Pull request merged**